README.md

```diff
@@ -1,32 +1,43 @@
-# @inrupt/solid-client
+# Solid JavaScript Client - solid-client-js
+@inrupt/solid-client is a JavaScript library for accessing data and managing permissions on data stored in Solid Pods. It provides an abstraction layer on top of both Solid and Resource Description Framework (RDF) principles and is compatible with the RDF/JS specification. You can use solid-client in Node.js using CommonJS modules and in the browser with a bundler like Webpack, Rollup, or Parcel.
 
-# Installation
-
-```bash
-npm install @inrupt/solid-client solid-auth-client
-```
+# Inrupt Solid JavaScript Client Libraries
+Inrupt Solid JavaScript Client Libraries are a group of libraries designed to support developers building Solid applications. solid-client provides an abstraction layer on top of both Solid and Resource Description Framework (RDF) principles and is compatible with the RDF/JS specification.
```

Comment:
This last sentence seems redundant with the previous paragraph?

```suggestion
Inrupt Solid JavaScript Client Libraries are a group of libraries designed to support developers building Solid applications.
```

(It's also weird to me to have "Inrupt" part of the name - "Inrupt's" would've made more sense to me. But I suppose that's a marketing decision?)